charts/bitwardenrs/1.1.1/values.yaml

```diff
@@ -0,0 +1,178 @@
+# Default values for bitwardenrs.
+
+replicaCount: 1
+
+image:
+  repository: bitwardenrs/server
+  pullPolicy: IfNotPresent
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+# bitwardenrs:
+#   domain: ""
+#   signupsAllowed: false
+#   gui:
+#     # If you set a different port here, you must also provide it under env
+#     port: 80
+#   websockets:
+#     enabled: true
+#     port: 3012
+#   admin:
+#     enabled: true
+#     disableAdminToken: false
+#     existingSecret:
+#       enabled: false
+#       name: ""
+#       tokenKey: ""
+#   # External database configuration.
+#   # Requires bitwardenrs/server >= 1.17.0 or bitwardenrs/server-{mysql,postgres} images
+#   # ref: https://github.com/dani-garcia/bitwarden_rs/wiki/Using-the-MySQL-Backend
+#   #      https://github.com/dani-garcia/bitwarden_rs/wiki/Using-the-PostgreSQL-Backend
+#   externalDatabase:
+#     enabled: false
+#     # Supported values: 'mysql', 'postgresql'.
+#     type: ""
+#     # Database host. Required if external database is enabled.
+#     host: ""
+#     # Database port. Optional, default value is specific to the database backend.
+#     port: ""
+#     # Database name.
+#     database: ""
+#     # Database user.
+#     user: ""
+#     # Database password. Special characters must be escaped with percent encoding.
+#     password: ""
+#     # Use existing secret for database credentials.
+#     existingSecret:
+#       enabled: false
+#       name: ""
+#       userKey: ""
+#       # Special characters in the password value must be escaped with percent encoding.
+#       passwordKey: ""
+#   # Enable SMTP. https://github.com/dani-garcia/bitwarden_rs/wiki/SMTP-configuration
+#   smtp:
+#     enabled: false
+#     # SMTP hostname, required if SMTP is enabled
+#     host: ""
+#     # SMTP sender e-mail address, required if SMTP is enabled
+#     from: ""
+#     # SMTP sender name, defaults to 'Bitwarden_RS'
+#     fromName: ""
+#     # Enable SSL connection
+#     ssl: true
+#     # SMTP port
+#     port: 587
+#     # SMTP username
+#     user: ""
+#     # SMTP password. Required is user is specified, ignored if no user provided
+#     password: ""
+#     # Use existing secret for SMTP authentication
+#     existingSecret:
+#       enabled: false
+#       name: ""
+#       userKey: ""
+#       passwordKey: ""
+#   # Enable Yubikey 2FA: https://github.com/dani-garcia/bitwarden_rs/wiki/Enabling-Yubikey-OTP-authentication
+#   yubico:
+#     enabled: false
+#     # OTP verification server. Will use the default YubiCloud servers if not specified
+#     server: ""
+#     # API Client ID for OTP server. Ignored if existingSecret is provided.
+#     clientId: ""
+#     # API Secret Key for OTP server. Required if clientId is specified, ignored when using existingSecret.
+#     secretKey: ""
+#     # Use existing secret for API keys
+#     existingSecret:
+#       enabled: false
+#       name: ""
+#       clientIdKey: ""
```

Comment:
I don't think the whole "bitwarden" section is going to work correctly.
k8s-at-home uses this to make the bitwarden environment variables (settings) easier.

But as we use questions.yaml, we don't actually need this at all. Because we can create our own labels in questions.yaml for every env-var from bitwarden. 

However, there are also some secrets here... I know k8s-at-home crafts those on demand, i'll need to look into that personally because we should do so from the common-chart instead of bitwarden-rs chart